models/survey/src/index.ts

```diff
@@ -14,25 +14,146 @@
 //
 
 import { type Builder } from '@hcengineering/model'
-
+import { type Class, type Ref } from '@hcengineering/core'
 import core from '@hcengineering/model-core'
 import workbench from '@hcengineering/model-workbench'
-import { surveyId } from '@hcengineering/survey'
+import {
+  type QuestionData,
+  type QuestionDataEditor,
+  type QuestionDataEditorComponentTypeRef,
+  surveyId
+} from '@hcengineering/survey'
 import survey from './plugin'
+import view from '@hcengineering/model-view'
+import {
+  TCheckboxes,
+  TInfo,
+  TQuestion,
+  TQuestionData,
+  TQuestionDataEditor,
+  TRadioButtons,
+  TSurvey,
+  TTypeRank
+} from './types'
 
+export { surveyOperation } from './migration'
 export { surveyId } from '@hcengineering/survey'
 export { default } from './plugin'
 
+// TODO: Should we place it into plugin itself, e.g. survey.routingParts.surveyId?
```

Comment:
I'm definitely don't like routes or route parts hardcoded as raw strings all over the place, when you can't Ctrl+Click it, find all usages or refactor. So enum or string constants are must. The only question is how we should expose it to other packages. I suggest make this decision later, once we need it outside for the very first time.